packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -167,6 +167,30 @@ abstract class WebViewPlatformController {
     throw UnimplementedError(
         "WebView getTitle is not implemented on the current platform");
   }
+
+  /// Sets the initial scale of page content.
+  ///
+  /// Only supported on Android.
+  ///
+  /// [scaleInPercent] must be between 0 and 100 inclusive.
+  ///
+  /// The behavior for the default initial scale depends on whether the WebView
+  /// supports the "viewport" HTML meta tag, whether the WebView is set to use a
```

Comment:
How do I know if my webview supports the viewport meta tag? how do I know if I'm set to use a wide viewport? what's controlling overview mode?

Reply:
These values are a part of Android [WebSettings](https://developer.android.com/reference/android/webkit/WebSettings). 
https://developer.android.com/reference/android/webkit/WebSettings.html#getUseWideViewPort()
https://developer.android.com/reference/android/webkit/WebSettings.html#getLoadWithOverviewMode()

I wanted to explain what the default behavior was without needing to add multiple other methods or refer to Android classes.

I decided against adding `boolean useWideViewPort` and `boolean getLoadWithOverviewMode` as optional parameters to `setInitialScale`, because I didn't want to expand the scope of this method in a critical issue PR.

Reply:
These are just the questions I had in mind when I read this documentation, we should answer these in the doc to the best of our ability.